### PR TITLE
Renaming intervention enum names

### PR DESCRIPTION
**Pull request by ghop02**

### CODE REVIEW

File: api/README.V1.md [hunk 1]
```diff
@@ -35,24 +35,25 @@ Specify either:
...
+"MODERATE_INTERVENTION"     // Social Distancing
+"STRONG_INTERVENTION"       // Stay at Home
+"OBSERVED_INTERVENTION"     // Dynamic forecast based on observations
 ```
 
 Additionally the most appropriate static scenario based on the per-state intervention is returned by specifying:
 
 ```js
-"SELECTED_MITIGATION"
+"SELECTED_INTERVENTION"
```
Comment (mikelehen) [inline api/README.V1.md:47]: remove

File: api/README.V1.md [hunk 2]
```diff
@@ -35,24 +35,25 @@ Specify either:
...
-"NO_MITIGATION",          // No Intervention
-"MODERATE_MITIGATION"     // Social Distancing
-"HIGH_MITIGATION"         // Stay at Home
+"NO_INTERVENTION",          // No Intervention
+"MODERATE_INTERVENTION"     // Social Distancing
+"STRONG_INTERVENTION"       // Stay at Home
+"OBSERVED_INTERVENTION"     // Dynamic forecast based on observations
 ```
 
 Additionally the most appropriate static scenario based on the per-state intervention is returned by specifying:
```
Comment (mikelehen) [inline api/README.V1.md:44]: I think we want to just remove this for now

File: api/README.V1.md [hunk 3]
```diff
@@ -147,20 +148,20 @@ This is the data format for both states and counties. `timeseries` is only inclu
...
   countyName, // null for states
   fips, // 2 digit for states, 5 digit for counties
-  lat, 
+  lat,
   long,
   lastUpdatedDate, // ISO 8601 date string
   actuals: {
     population,
-    intervention, // one of (NO_MITIGATION, MODERATE_MITIGATION, stay_at_home)
+    intervention, // one of (NO_INTERVENTION, MODERATE_INTERVENTION, OBSERVED_INTERVENTION, SELECTED_INTERVENTION)
```
Comment (mikelehen) [inline api/README.V1.md:156]: Remove SELECTED_INTERVENTION?
Are we missing STRONG_INTERVENTION?

File: api/README.V1.md [hunk 4]
```diff
@@ -35,24 +35,25 @@ Specify either:
 Forward projections are available for the following scenarios:
 
 ```js
-"NO_MITIGATION",          // No Intervention
-"MODERATE_MITIGATION"     // Social Distancing
-"HIGH_MITIGATION"         // Stay at Home
+"NO_INTERVENTION",          // No Intervention
+"MODERATE_INTERVENTION"     // Social Distancing
```
Comment (mikelehen) [inline api/README.V1.md:39]: Should be WEAK_INTERVENTION here and elsewhere.

File: api/README.V1.md (hunk 3 above)
Comment (ghop02) [reply api/README.V1.md:156]: SELECTED is still an option here so feels like we should surface it?

**Pull request merged**